cypress/e2e/tests/administration/jira-connection/sort.test.ts

```diff
@@ -54,7 +53,7 @@ describe(["@tier2"], "Jira connections sort validations", function () {
             !useTestingAccount
         );
         jiraConnectionList.forEach((jira) => {
-            jira.create(!toBeCanceled, expectedToFail);
+            jira.create(!toBeCanceled);
```

Comment:
tobeCancelled = false by default so it should not be passed when we are not cancelling.
jira.create() should be able to create JIRA instance.

Reply:
How come that his file got to this PR at all?

Reply:
Which file ?

Reply:
Fixed